src/containers/Accounts/AMM/AMMAccounts/AMMAccountHeader/AMMAccountHeader.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import 'containers/shared/css/nested-menu.scss'
+import 'containers/Accounts/AccountHeader/styles.scss'
+import 'containers/Accounts/AccountHeader/balance-selector.scss'
+import { AmmDataType } from '../index'
+import { localizeNumber, localizeBalance } from '../../../../shared/utils'
+import Currency from '../../../../shared/components/Currency'
+
+export const AMMAccountHeader = (props: AmmDataType) => {
+  const {
+    ammId,
+    balance,
+    balance2,
+    tradingFee,
+    lpBalance,
+    accountId,
+    language,
+  } = props
+  const { t } = useTranslation()
+  const b1 = localizeBalance(balance, language)
+  const b2 = localizeBalance(balance2, language)
+  const tradingFeeTotal = 1000
+  const lp = localizeNumber(lpBalance, language, {
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 2,
+  })
+  const tf = localizeNumber(tradingFee / tradingFeeTotal, language, {
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 3,
+  })
+
+  function renderHeaderContent() {
+    return (
+      <div className="section header-container">
```

Comment:
@ckniffen as a future task, maybe the page headers should all be unified.

Reply:
That is in the plans.